### PR TITLE
Corrected the content.

### PR DESCRIPTION
**Pull request by Meruja**

### CODE REVIEW

File: en/docs/install-and-setup/upgrading-wso2-api-manager/upgrading-from-250-to-320.md [hunk 1]
```diff
@@ -2611,7 +2611,7 @@ Follow the instructions below to configure WSO2 API Manager for the WSO2 API-M A
         password = "password"
         ```
 
-    -   The second datasource points to the **WSO2 Data Analytics (WSO2 DAS) based previous datasource WSO2AM_STATS_DB for statistics** .
+    -   The second datasource points to the **WSO2 Data Analytics (WSO2 DAS) based newly migrated WSO2AM_STATS_DB for statistics**.
```
Comment (shilmyhasan) [inline en/docs/install-and-setup/upgrading-wso2-api-manager/upgrading-from-250-to-320.md:2614]: This chane is not required , as we use the term stat db only for previous versions analytics database and it should be the previous versions datasouce. The issue is only in AM datasource

Comment (Meruja) [reply en/docs/install-and-setup/upgrading-wso2-api-manager/upgrading-from-250-to-320.md]: Fixed with 6e72fda

**Review by shilmyhasan — APPROVED** [review]

File: en/docs/install-and-setup/upgrading-wso2-api-manager/upgrading-from-250-to-320.md [hunk 2]
```diff
@@ -2601,7 +2601,7 @@ Follow the instructions below to configure WSO2 API Manager for the WSO2 API-M A
 
     The following in an example of how the configurations should be defined when using MySQL.
 
-    -   The first datasource points to the **previous API-M version's AM_DB datasource.**
+    -   The first datasource points to the **newly migrated AM_DB datasource.**
```
Comment (shaniR) [inline en/docs/install-and-setup/upgrading-wso2-api-manager/upgrading-from-250-to-320.md:2604]: why can't we use the version?

Comment (Meruja) [reply en/docs/install-and-setup/upgrading-wso2-api-manager/upgrading-from-250-to-320.md]: Fixed with cf7405e

**Review by Mariangela — APPROVED** [review]

**Comment by Mariangela** [comment]
@Meruja , is a similar change applicable to the 3.1.0 and 3.0.0 doc versions? Also when sending fixes to 3.2.0 always send it to the master as well.

**Comment by Meruja** [comment]
> @Meruja , is a similar change applicable to the 3.1.0 and 3.0.0 doc versions? Also when sending fixes to 3.2.0 always send it to the master as well.

No, we don't need to do this fix for  3.1.0 and 3.0.0 doc versions. The fix for master has been sent. https://github.com/wso2/docs-apim/pull/2268

**Pull request merged**